tensorrt_llm/_torch/modules/fla/l2norm.py

```
@@ -0,0 +1,152 @@
+            num_warps=8,
+            num_stages=3,
+        )
+
+    return y.view(x_shape_og)
+
+
+class L2NormFunction(torch.autograd.Function):
+
+    @staticmethod
+    @input_guard
+    def forward(ctx, x, eps=1e-6, output_dtype=None):
```

Ruff (0.13.1)

Unused static method argument: `ctx`

(ARG004)